### ISSUE TITLE
Speaker Notes are not rendered on Speaker mode

### ISSUE THREAD

**Issue by anarute** [hedgedoc/hedgedoc#3862]
### Description

When starting the Speaker mode of a presentation the Notes don't appear.

### Steps to reproduce

1. Create a note with the slide type
2. Go to Menu > Slide Mode
3. Press "S" to display the Speaker mode

Any slide that should contain note displays only the title "Notes" but not the actual note.

### Expected behaviour

That the notes would appear in the Speaker mode.

### Logs

In the console of the speaker window I can see the following errors:

```js
Uncaught ReferenceError: exports is not defined
    at marked.js:30:802

notes.html:388 Uncaught TypeError: pendingCalls[data.callId] is not a function
    at notes.html:388:33

Uncaught ReferenceError: marked is not defined
    at handleStateMessage (notes.html:455:8)
    at notes.html:836:11
    at notes.html:385:8
```

### Your Setup

This is happening using the docker image, for example: https://demo.hedgedoc.org/p/ykfU7WlJk#/

When installing and running locally with the setup below it works perfectly
- Host OS: Fedora 37
- NodeJS version: 16.20.0
- HedgeDoc version: 1.9.7

**Comment by mrdrogdrog** [comment]
The demo instance also uses the docker image. So it doesn't seem like a problem with the docker image.

**Comment by anarute** [comment]
> The demo instance also uses the docker image. So i'm not sure if the problem is the docker image.

Thanks for replying, yes I'm trying to see if I find the exact setup where this is reproducible because building and running with docker from scratch works fine.. so I can only see the bug in the [demo instance](https://demo.hedgedoc.org/p/ykfU7WlJk#/) and in one of our hosted instances but not all of our instances. I wonder if it's a cached dependency or something like that, not sure yet.

**Comment by mrdrogdrog** [comment]
The pad link you've sent me works for me. Which browser did you use?

**Comment by anarute** [comment]
> The pad link you've added works for me. Which browser did you use?

I tried in Chromium and Firefox, the speaker window shows but not the note:

![Screenshot from 2023-04-25 09-49-31](https://user-images.githubusercontent.com/333447/234210414-6e5b8acf-0d4b-420e-8230-5701221b127b.png)

There should be a  "Test" written there https://demo.hedgedoc.org/9KTr9xB-Qliv6DVeuyL7Ww?both

**Comment by mrdrogdrog** [comment]
Ohhhh. Now I get it. I'm sorry for misunderstanding you. :sweat_smile: 

That looks like a problem with the production build of the application. I'll take a look. :)

**Comment by anarute** [comment]
Btw, we just noticed that using the `aside` element with the class `notes` is working

```
<aside class="notes">
test
</aside>
```

Example: https://demo.hedgedoc.org/9KTr9xB-Qliv6DVeuyL7Ww?both

**Comment by ErikMichelson** [comment]
This has been fixed with the 1.9.8 release.
